extensions/wikia/UserLogin/UserLoginHelper.class.php

```diff
@@ -133,97 +133,82 @@ protected function addUserToUserList( $userId, &$userList ) {
 	}
 
 	/**
-	 * get a list of random popular wikis
-	 * @param integer $require (number of wikis)
-	 * @return array $wikis
+	 * Redirect the user to the appropriate page after login.
+	 *
+	 * @requestParam string returnUrl
 	 */
-	public function getRandomWikis( $require=self::LIMIT_WIKIS ) {
```

Comment:
Got rid of this unused function after checking with Cristina on the functionality it was providing.  I guess Tor added this early return in 2012 while investigating a slow query here and never finished up.  Blame Tor!  I cleaned up everything above that was calling this.